.buildkite/scripts/resolve_launcher_actions.sh

```diff
@@ -16,7 +16,7 @@ promote_from_one_channel_to_another() {
     from_channel="${3}" # e.g. "stable"
     to_channel="${4}"   # e.g. "rc-0.75.0"
 
-    artifact="$(latest_from_builder "${target}" ${from_channel} ${package_name})"
+    artifact="$(latest_from_builder "${target}" $"{from_channel}" "${package_name}")"
```

Comment:
I don't know what is going here, I think this is NOT intended. @scotthain @baumanj 
```bash
FOO=BAR
echo $"{FOO}"
${FOO}
echo "${FOO}"
BAR
```

Reply:
@jsirex Well spotted; this got fixed in https://github.com/habitat-sh/habitat/pull/6219